### ISSUE TITLE
Conversion Liter <--> Gallon incorrect

### ISSUE THREAD

**Issue by JamesK235** [hgrecco/pint#336]
I am trying to convert liters to gallons. 4 gallons should equal 1.05669 gallons, however when I convert 4 liters it equals 1.32086 gallons.

```
>>> import pint
>>> ureg = pint.UnitRegistry()
>>> volume = 4 * ureg.L
>>> volume.to('gallon')
<Quantity(1.3208602617907421, 'gallon')>
```

When I use the unit registry in default_en_0.6.txt, it converts correctly 4 liters to 1.05669 gallons. When I use pint 0.7 or 0.8, it is incorrect.

Thank you for your time and assistance.

**Comment by hgrecco** [comment]
Thanks for reporting. There is an error in the definitions file. I will fix it and release 0.7.1 tonight. Sorry for the inconvenience.

**Comment by hgrecco** [comment]
Fixed
